<launch>
  <arg name="input_signal" default="/roi_signal_proj"/>
  <arg name="input_image" default="/image_raw"/>

  <node name="feat_proj_bbox"
        pkg="nodelet" type="nodelet"
        output="screen"
        args="standalone trafficlight_recognizer/FeatProjBBox">
    <remap from="~input_signal" to="$(arg input_signal)" />
    <remap from="~input_image" to="$(arg input_image)" />
    <rosparam subst_value="true">
      z_max: 100
      z_min: 20
      w_max: 350
      w_min: 50
      h_max: 200
      h_min: 50
      approximate_sync: true
    </rosparam>
  </node>

</launch>
